custom_components/tuya_local/devices/dr_heater_dr008_thermostat.yaml

```diff
@@ -0,0 +1,229 @@
+      - id: 108
+        type: boolean
+        name: lock
+        mapping:
+          - dps_val: true
+            icon: "mdi:hand-back-right-off"
+          - dps_val: false
+            icon: "mdi:hand-back-right"
+  - entity: sensor
+    name: Room temperature
+    class: temperature
+    icon: "mdi:home-thermometer"
+    dps:
+      - id: 105
+        type: integer
+        name: sensor
+        class: measurement
+        mapping:
+          - scale: 10
+      - id: 107
+        name: unit
+        type: boolean
+        mapping:
+          - dps_val: false
+            value: C
+          - dps_val: true
+            value: F
+  - entity: sensor
+    name: Floor temperature
+    class: temperature
+    icon: "mdi:thermometer-lines"
+    dps:
+      - id: 106
+        type: integer
+        name: sensor
+        class: measurement
+        mapping:
+          - scale: 10
+      - id: 107
+        name: unit
+        type: boolean
+        mapping:
+          - dps_val: false
+            value: C
+          - dps_val: true
+            value: F
+  - entity: sensor
+    name: Energy
+    dps:
+      - id: 117
+        type: integer
+        name: sensor
+        unit: kWh
+        mapping:
+          - scale: 10
+      - id: 116
+        type: integer
+        name: calibration
+  - entity: select
+    category: config
+    translation_key: temperature_unit
+    dps:
+      - id: 107
+        name: option
+        type: boolean
+        mapping:
+          - dps_val: false
+            value: celsius
+          - dps_val: true
+            value: fahrenheit
+  - entity: number
+    category: config
+    class: temperature
+    name: Room temperature limit
+    icon: "mdi:thermometer-chevron-down"
+    dps:
+      - id: 121
+        type: integer
+        name: value
+        range:
+          min: 200
+          max: 500
+        mapping:
+          - scale: 10
+            step: 5
+            constraint: unit
+            conditions:
+              - dps_val: true
+                step: 30
+                range:
+                  min: 680
+                  max: 1220
+      - id: 107
+        name: unit
+        type: boolean
+        mapping:
+          - dps_val: false
+            value: C
+          - dps_val: true
+            value: F
+  - entity: select
+    category: config
+    name: Temperature sensor
+    icon: "mdi:thermometer-check"
+    dps:
+      - id: 111
+        type: string
+        name: option
+        mapping:
+          - dps_val: "0"
+            value: "Room"
+            icon: "mdi:home-thermometer"
+          - dps_val: "1"
+            value: "Floor"
+            icon: "mdi:heating-coil"
+          - dps_val: "2"
+            value: "Both"
+            icon: "mdi:thermometer"
+  - entity: select
+    category: config
+    name: Auto schedule
+    icon: "mdi:calendar-sync"
+    dps:
+      - id: 110
+        type: integer
+        name: option
+        mapping:
+          - dps_val: 0
+            value: "7"
```

Comment:
Usual values used for this type of Schedule are "Daily", "Mon-Sat+Sun", "Weekday+Weekend". These indicate one schedule for every day, one schedule for Mon-Sat and a different schedule for Sunday, and one schedule Mon-Fri and a different schedule for Sat-Sun, respectively.

But your "Adaptive" and also the "5+1+1" seems to suggest something slightly different than the normal "7", "6+1", "5+2" from Tuya. But anyway maybe there is wording similar to the above that will make the meaning clearer.

Reply:
True, this does appear different than normal, but it matches what is shown in the Tuya app
![Screenshot_20241207-212752](https://github.com/user-attachments/assets/0d9f9c62-a392-4964-a822-e461a25d0b18)